packages/plugin-board/src/realtime.js

```diff
@@ -82,9 +82,47 @@ const RealtimeService = Mercury.extend({
     if (contentType) {
       data.data.contentType = contentType;
     }
+
+    // use mercury socket if it is shared
+    // if this.socket is defined then use it instead
+    const isSharingMercuryFeatureEnabled = this.spark.device.features.developer.get(`web-shared-mercury`);
+    if (isSharingMercuryFeatureEnabled && isSharingMercuryFeatureEnabled.value && !this.socket) {
+      return this.spark.mercury.socket.send(data);
+    }
+
     return this.socket.send(data);
   },
 
+  /**
+   * Connect and use an exisiting mercury connection
+   * @memberof Board.RealtimeService
+   * @param  {Board~Channel} channel
+   * @returns {Promise<Board~Registration>}
+   */
+  connectToSharedMercury(channel) {
```

Comment:
Did this logic move into the sdk yet? I don't see anything reading sharedWebSocket.